client/src/components/Link/styles.module.scss

```
@@ -4,16 +4,32 @@
     margin: 0;
     padding: 0;
     position: relative;
-    font-size: 44px;
+    font-size: 2.2em;
```

I normally use rem. Why did you decide to use em?

```suggestion
    font-size: 2.2rem;
```

---

cause I want it to be related to the font of parent element, not of the root element